examples/tutorial/jupyter/execution/pandas_on_dask/requirements.txt

```diff
@@ -1,5 +1,4 @@
 fsspec
-s3fs
```

Comment:
did you search for all files listing `s3fs` dependency? what about `env_omnisci.yml` or other requirement files?

Reply:
We use `s3fs` for testing here: https://github.com/modin-project/modin/blob/adb16a17f721048005520388080627975c6852d8/modin/pandas/test/test_io.py#L1527

By this reason `s3fs` wasn't removed from dev-env files, but removed from other (like example deps), because modin package no longer requires `s3fs`